Observatory-Control-System/EAS/DIMM/ops.rst

```diff
@@ -106,8 +106,24 @@ and within the OpenTPL:
         AUTH OK 0 0
         1 set ameba.mode=1
         
+It is common to see ``EVENT WARN`` messages regarding the state of ameba. This does not necessarily indicate an issue.
 
-Verify the DIMM CSC is enabled and data is appearing in the EFD.
+The status of ameba can be checked with with following command to *tt-master*. This can be useful to check if the DIMM will operate tonight, or if there were concerning warning messages when starting ameba.
```

Comment:
In reST we write one sentence per line for easier diff and reading: The sentence starting with "This can" can be moved to the line 112, but it will be still interpret as the same paragraph. To start a new paragraph we leave a blank line in between.